src/security/PermissionsPolicy.ts

```diff
@@ -13,18 +13,43 @@ class PermissionsPolicy extends Test {
 
     if (!Object.prototype.hasOwnProperty.call(response.headers, 'permissions-policy')) {
       return {
-        status: 'WARNING',
+        status: 'ERROR',
         title: 'Permissions-Policy',
         description: 'Response headers does not contain permissions-policy header!',
       };
     }
 
+    const permissionsArray = ['accelerometer', 'geolocation', 'midi', 'notifications', 'push', 'sync-xhr', 
+    'microphone', 'camera', 'magnetometer', 'gyroscope', 'speaker', 'vibrate', 'fullscreen', 'payment', 'usb'];
+
+    const attributesList = response.headers['permissions-policy'];
+    let subChecks = [];
+    subChecks = this.checkPermission(permissionsArray, attributesList);
+
     return {
-      status: 'SUCCESS',
+      status: subChecks.some(check => check.status === 'WARNING') ? 'WARNING' : 'SUCCESS',
       title: 'Permissions-Policy',
-      description: `The value of permissions-policy header is ${response.headers['permissions-policy']}.`,
+      description: '',
+      results: subChecks,
     };
   }
+  
+  private checkPermission(permissionsArray: string[], attributes: string): Result[] {
```

Comment:
This is checking an arrray of permissions, so would be better to name it `checkPermissions`.